src/distributionPaybackMessage.js

```diff
@@ -0,0 +1,22 @@
+const { promises: fs } = require('fs')
+const Discord = require("discord.js")
+const client = new Discord.Client()
+const addressBook = require("./addressbook.json")
+const daoAddress = 'theDAOaddress'
+
+async function paybackMessage() {
+    let contents = await fs.readFile('./owed.csv', {encoding: 'utf8'})
+    contents = contents.trim().split("\n")
+
+    for (content of contents) {
+        let [address, balanceOwed] = content.split(",")
+        let user = addressBook.find(function findUser(user) {
+            return user.address === address
+        })
+
+        console.log(user.discordId)
```

Comment:
Discord.js has a `client.guild.members.cache.get(discordId)` function that returns Discords User object. You can then send a message with user.send()